### PR TITLE
Setup 100

### PR DESCRIPTION
**Pull request by mofirojean**
Created the set-up module and the also added the organization entity and payload

### CODE REVIEW

File: src/app/models/entity/organisation/organisation.entity.ts [hunk 1]
```diff
@@ -0,0 +1,7 @@
+import { BaseEntity } from "../base/base.entity";
+
+export class OrganisationEntity extends BaseEntity {
+    constructor(public name: string) {
+        super();
+    }
+}
```
Comment (elroykanye) [inline src/app/models/entity/organisation/organisation.entity.ts:7]: move this `organisation.entity.ts` file into the school folder instead

File: src/app/modules/application/setup/components/org-setup-organisation/org-setup-organisation.component.ts [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+import {OrganisationPayload} from "../../../../../models/payload/organisation.payload";
+
+@Component({
+  selector: 'app-org-setup-organisation',
+  templateUrl: './org-setup-organisation.component.html',
+  styleUrls: ['./org-setup-organisation.component.css']
+})
+export class OrgSetupOrganisationComponent {
+
+  submit = (org: OrganisationPayload) => {
```
Comment (elroykanye) [inline src/app/modules/application/setup/components/org-setup-organisation/org-setup-organisation.component.ts:11]: complete the logic for this method. you have to write the service and call the endpoint

File: src/styles.css [hunk 3]
```diff
@@ -4,3 +4,8 @@
 @import "node_modules/primeng/resources/primeng.min.css";
 @import "node_modules/primeflex/primeflex.min.css";
 @import "node_modules/primeicons/primeicons.css";
+
+/* commoly used color */
+.color-blue {
+    color: #0077b6 !important;
+  }
```
Comment (elroykanye) [inline src/styles.css:11]: add an empty line to end of every file. i do not really know why but it is a good thing

**Review by elroykanye — CHANGES_REQUESTED** [review]

File: src/styles.css (hunk 3 above)
Comment (mofirojean) [reply src/styles.css:11]: Ok Kanye noted

File: src/app/modules/application/setup/components/org-setup-organisation/org-setup-organisation.component.html [hunk 4]
```diff
@@ -0,0 +1,3 @@
+<div class="surface-section border-round-xl px-3 py-2">
+    <app-org-setup-organisation-form (submitEvent)="submit($event)"></app-org-setup-organisation-form>
```
[the remark this answers is not in the document]
Comment (mofirojean) [reply src/app/modules/application/setup/components/org-setup-organisation/org-setup-organisation.component.html:2]: Will correct that as well

**Review by elroykanye — APPROVED** [review]

**Pull request merged**